app/celery/scheduled_tasks.py

```diff
@@ -418,12 +418,13 @@ def zendesk_new_email_branding_report():
         brands_with_no_organisation=brands_with_no_organisation,
     )
 
-    ticket = NotifySupportTicket(
-        subject="Review new email brandings",
-        message=message,
-        ticket_type=NotifySupportTicket.TYPE_TASK,
-        technical_ticket=False,
-        ticket_categories=["notify_no_ticket_category"],
-        message_as_html=True,
-    )
-    zendesk_client.send_ticket_to_zendesk(ticket)
+    if current_app.config["NOTIFY_ENVIRONMENT"] in ["live", "production", "test"]:
```

Comment:
nit: could be a set 😉

Reply:
actually i might go and do the decent thing and make a nice model thing

Reply:
fun fact: im pretty sure that `live` _and_ `production` being in the list is because we just weren't sure what the variable would be on prod paas 🐵

Reply:
yeah I figured that might be the case 😂